content/applications/finance/accounting/get_started/chart_of_accounts.rst

```diff
@@ -3,48 +3,37 @@ Chart of accounts
 =================
 
 The **chart of accounts (COA)** is the list of all the accounts used to record financial
-transactions in the general ledger of an organization.
+transactions in the general ledger of an organization. The chart of accounts can be found under
+:menuselection:`Accounting-->Configuration-->Chart of Accounts`.
 
-The accounts are usually listed in the order of appearance in the financial reports. Most of the
-time, they are listed as follows :
+When browsing your chart of accounts, you can notably filter the accounts by :guilabel:`Code`,
+:guilabel:`Account Name`, or :guilabel:`Type`, but other options are available in the dropdown menu
```

Comment:
FYI there's dropdown and drop-down spelling in the same doc, I'd choose just one ;)